### ISSUE TITLE
Doctype as sentence

### ISSUE THREAD

**Issue by andrenatal** [mozilla/voice-web#300]
And the user actually pronounced it.

<img width="795" alt="screenshot 2017-07-19 12 34 17" src="https://user-images.githubusercontent.com/973388/28385967-e1ca9efe-6c7e-11e7-8f67-eb9ba8d2ef34.png">

**Comment by mbebenita** [comment]
I'd love to hear that.

**Comment by cherls** [comment]
That should be my voice. I did a number of those yesterday when the website was bugging out. Another instance I read was `<head><title>`. 

However, I did not read out the special characters (e.g. "open bracket", "exclamation mark") — only the pronounceable words or letters. In retrospect, I wish I did as that as it could be a specific use case of speech to text; blind programmers for example.

**Comment by mbebenita** [comment]
That's an interesting idea that I hadn't crossed my mind. Training STT models for programmers.

**Comment by mikehenrty** [comment]
Yup, this was definitely caused by #274, and now we have some garbage recordings in the DB. The only fix for this would not be in code, but would be to go into the database and remove all these. Not sure if it's worth it, but we should at least find out how many recordings look like this.

**Comment by rugk** [comment]
> "open bracket"

You also would have to say ["open angle bracket"](https://en.wiktionary.org/wiki/angle_bracket). ONly this would be correct, as a "usual" bracket is `(`.
